spring-session-data-mongodb/src/integration-test/java/org/springframework/session/data/mongo/MongoDbDeleteJacksonSessionVerificationTest.java

```diff
@@ -142,14 +143,10 @@ static class SecurityConfig {
 		@Bean
 		SecurityWebFilterChain securityWebFilterChain(ServerHttpSecurity http) {
 			return http //
-					.logout()//
-					/**/.and() //
-					.formLogin() //
-					/**/.and() //
-					.csrf().disable() //
-					.authorizeExchange() //
-					.anyExchange().authenticated() //
-					/**/.and() //
+					.logout(Customizer.withDefaults())//
+					.formLogin(Customizer.withDefaults()) //
+					.csrf((csrf) -> csrf.disable()) //
+					.authorizeExchange((ae) -> ae.anyExchange().authenticated()) //
```

Comment:
We can probably remove those `//` too and wrap the block of code with `// @formatter:off` and `// @formatter:on`

```java
// @formatter:off
return http 
					.logout(Customizer.withDefaults())
					.formLogin(Customizer.withDefaults()) 
					.csrf((csrf) -> csrf.disable()) 
					.authorizeExchange((ae) -> ae.anyExchange().authenticated())
                     .build();
// @formatter:on
```

Reply:
done.